vulnerabilities/scraper/npm.py

```diff
@@ -0,0 +1,105 @@
+# Author: Navonil Das (@NavonilDas)
+import semantic_version
+import re
+import json
+from urllib.request import urlopen
+
+NPM_URL = 'https://registry.npmjs.org{}'
+PAGE = '/-/npm/v1/security/advisories?page=1'
+    
+# Removes spaces and v Charecter in front of version 
+def remove_spaces(x):
+    x = re.sub(r" +"," ",x)     # Replace Multiple spaces to one
```

Comment:
Please avoid end of line comments if possible. And use single ' quotes unless for docstrings.

That said, what is the purpose of this function? Are spaces an issue for the semantic version library?
And this is a rather complex function, so we absolutely need many tests for this